crt_portal/cts_forms/templates/forms/complaint_view/disposition/actions/batch/batch_actions.html

```diff
@@ -0,0 +1,46 @@
+{% extends "forms/complaint_view/show/card.html" %}
+
+{% load i18n %}
+
+{% block title %}Batch #{{ batch.uuid }}{% endblock %}
+
+{% block extra_classes %}disposition-actions-card crt-actions-card{% endblock %}
+
+{% block card_content %}
+
+<form class="usa-form" method="post" action="/form/disposition/batch" novalidate>
+    {% csrf_token %}
+    <input type="hidden" value="{{ batch.uuid }}" name="uuid" id="uuid" />
+    <div>
+        <h3>Select reports to approve for disposal</h3>
+        <a class="text-bold" href="#">View Proposed Reports for Disposal</a>
```

Comment:
suggestion: I find it helpful to add a TODO(#issueNumber) so that I can search for the places I left missing functionality

Reply:
This is no longer needed so I remove it entirely in the next PR but yes noted for future changes.